Medsci remap strikes back

S34NW opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do

Revives most of https://github.com/ParadiseSS13/Paradise/pull/15064

Things have changed since that PR, since we didn't have agreed upon standards back then I had a LOT of random crap to touch up and tidy. There is a lot to digest here so I need some mapper eyes to make sure I've not missed anything we currently enforce.

I'm also happy to take on any feedback and am more than willing to change things, as the map was not guided by my vision I won't be hesitant to make changes if they are not garbage.
<!-- Include a small to medium description of what your PR changes. Document all changes, as not doing this may delay reviews or even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
This was a community favourite that fell by the wayside due to previous leadership being paralysed and unable to tackle the map in a reasonable timeframe.

Science maints especially was in need of some TLC, it was a linear corridor full of of pain and bad mapping practices. This tidies that.
Medbay has been split across multiple rooms instead of one single general purpose "come here and be automender'd" room.

Genetics moved into science on box (accesses and actual assignment unchanged) as they don't really do anything with cloning on box.

<!-- Add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Images of changes

![2022 08 04-15 28 49](https://user-images.githubusercontent.com/12197162/182872994-33d58856-bd65-42f8-a4ef-432e12a0881c.png)

<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif or mp4 of your feature if you want. -->

## Changelog
:cl: FreeStylaLT, S34N
tweak: Cyberiad Medbay/Science/Escape/South-East maint remap
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->
<!-- If a PR has no impact on players (i.e. a code refactor that does not change functionality) then the entire Changelog heading and contents can be removed. -->

S34NW commented:
oh god the tag vars

BakKeener commented:
Virology having windows in maint doesn't seem very safe, especially since I last recall... sneezing goes through windows and coughs too probably unless that got patched.

S34NW commented:
Please react 👍🏼 or 👎🏼 on this comment if you think maint vents for this large maint area is worth keeping.

BakKeener commented:
> Please react 👍🏼 or 👎🏼 on this comment if you think maint vents for this large maint area is worth keeping. (They are on their own subnet separate from distro, serviced by the mini-atmos room south of viro)

I think this will finally shake up the mid round blob meta instead of just turbine and elect maint so it'll make those rounds much more chaotic and fun while also helping our terrors with some fast maint movement in sci maint making med/sci maint a truly dangerous hotspot for non antags to be in when stuff is going down.

Komrad822 commented:
The window in toxins storage, either it becomes plasma window or gets removed entirely. SM has plasma windows to contain hazards that occur in the reactor (Rads, heat, pressure, anomalies, etc.), same should be done with a room full of cannisters containing hazardous canisters. 
But a counter argument is that it gives terrors an advantage cause they can start the fire in toxins storage and it break the window and lead out instead of just fizzing out inside the storage as sometimes happens if terrors don't proactively prepare for the fire before unleashing it.

S34NW commented:
> The window in toxins storage, either it becomes plasma window or gets removed entirely.

It was added specifically to allow gas spreading, previously walls would melt and spread but since that was removed tox storage has been basically leak-proof unless something bolts the door open and opens firedoors.

PacifistDalek commented:
YESSS IT RETURNS.

One thing i've been hoping for is another stash of atmos scrubbers/heaters, etc. There are loose heaters in maint but it'd be great to have an atmos room near escape that could have a few scrubbers and the like in it, so that you don't have to go all the way back to engineering to get one, especially since between toxins, scichem, and escape, there are a lot of usecases for scrubbers and the like. Keeping it behind an atmos door would keep science from just grabbing them pre-emptively from maint. (If i'm oblivious and there is a stash somewhere nearby, never mind!)

Sikroth commented:
Still no built in kill-chamber in xenobio? I feel like that should be a given by now. Having to place a shower surrounded by furniture every shift is jank.

Personally not a fan of the new medbay layout. Why would you *not* want a big open main treatment area? I can see myself removing the grass bits every shift and possibly placing the sleepers there, instead of spread around 3 different side rooms.

Do we really need to have a cloning-to-morgue disposals pipe? It feels really lazy and a way to remove what little interaction Coroners already get when someone bothers to enter the morgue to bring a body in. Same for the robotics and genetics ones. I'm sure geneticists could use the 10 seconds of walking to dispose of a monkey in their 2h of afk clicking buttons.

Another personal nitpick but I wish the security waiting room in departures didn't partially cut into the general departures room. Not really a big deal but it'd look better if departures was a square round without that sec bit biting into it, imho.

S34NW commented:
> Still no built in kill-chamber in xenobio? I feel like that should be a given by now. Having to place a shower surrounded by furniture every shift is jank.

Added, thanks for feedback.

lewcc commented:
medical wrench my beloved

A-Medster commented:
Im down to clown with these changes, liking the new look. I like the dedicated corpse shoot for robotics, science, and medical. Though one nitpick i have is the kill room: you're not really breeding slimes in there so theres really no need for a monkey shoot there

Miraviel reviewed (changes requested):
I will review different sections in different reviews, if that is alright, to keep these comments from growing too big. Starting with virology!

<details>
  <summary>Virology 1 - Newscaster</summary>

  This room could use a newscaster. The virologist spends most of their time in this room, completely isolated - let them have some fun!

  ![image](https://user-images.githubusercontent.com/33333517/180783668-3198739d-ca95-4f6c-b09b-0219034fa33e.png)
</details>

<details>
  <summary>Virology 2 - Two air alarms</summary>
  Virology has two air alarms, one in the pandemic room, one in the breakroom, controlling the same vents. Not sure if intended.
 
  
![image](https://user-images.githubusercontent.com/33333517/180784889-44a756b2-51bc-45b8-b070-53017762ab83.png)

</details>

<details>
  <summary>Virology 3 - Missing firelock?</summary>
  
  This airlock seems to be missing a firelock.

![image](https://user-images.githubusercontent.com/33333517/180784236-0c880a4b-554c-44c0-a4e8-9778c3956820.png)

</details>

<details>
  <summary>Virology 4 - Blood Crate</summary>
  
  This blood crate has different types of blood for the (I suppose) monkeys and some emergency tanks... But no masks. Since it is not intended(?) for the virologist, who has their O2 box, this crate could use two extra masks (preferably as a subtype of this freezer.)
  
![image](https://user-images.githubusercontent.com/33333517/180785161-14b29c2e-58de-4400-bea4-ebef47ee4cea.png)

</details>

<details>
  <summary>Virology 5 - No intercom</summary>

  No intercom in the break room, not sure if intended or just missing.
  
![image](https://user-images.githubusercontent.com/33333517/180785694-48e12f17-f684-45e1-b041-32b2340001ee.png)
</details>

<details>
  <summary>Virology 6 - Two intercoms</summary>
  
  The cycler has two intercoms.

![image](https://user-images.githubusercontent.com/33333517/180785864-bd7324d8-046f-4c30-a25a-413d37cc3b06.png)

</details>

<details>
  <summary>Virology 7 - Cycler button is on top of a poster</summary>

  It is a little hard to see at first because it is on top of the biohazard poster.  

![image](https://user-images.githubusercontent.com/33333517/180786815-8b619060-7707-4876-bbe2-efb218e62739.png)

</details>

<details>
  <summary>Virology 8 - Atmospherics</summary>
  
  In general, atmospherics seem to be incredibly spread out in this section, way too many rooms are handled only by one air alarm (similarly to cyberiad's old permabrig corridor). There are five rooms, a cycler, and a corridor that shares vents from another air alarm, this one:

![image](https://user-images.githubusercontent.com/33333517/180786250-1d7be0d1-578e-4570-bfe8-45e7365a1078.png)

  If you could split this up at least into two and/or you could add two airlocks to those firelocks so this corridor section is airtight, that would help future atmos techs figuring out what the heck is going on.

 Virology seems to get its own air from the room on its south, yet the air alarm controlling this section also handles the air coming from the main loop (here):

![image](https://user-images.githubusercontent.com/33333517/180786632-21ea2e8b-4ee5-4821-93b1-9177b92c67cc.png)


</details>


<details>
  <summary>Virology 9 - Camera and light in a corner</summary>
  
  This camera and light share the same tile and they are in a corner, they look awkward.
  
![image](https://user-images.githubusercontent.com/33333517/180787346-db79280d-a147-4a2c-b4a2-64d51026ab97.png)

</details>

Miraviel reviewed (changes requested):
Coroner review

<details>
  <summary>Coroner 1 - Interior door</summary>
  
  The windoor north of me is called "interior door" and has no access requirement.

  ![image](https://user-images.githubusercontent.com/33333517/180787850-c354c377-540f-41db-bdee-657e56325128.png)
  
</details>

<details>
  <summary>Coroner 2 - Corner Camera</summary>
  
  The camera is in the corner, corner cameras don't see much and look strange, might want to move it.

  ![image](https://user-images.githubusercontent.com/33333517/180788110-3ab3b0cc-97a0-4954-82e8-7780942b0399.png)

</details> 


<details>
  <summary>Coroner 3 - Unreachable desk</summary>
  
  The body bag boxes on the desk northwest of me cannot be reached.

 ![image](https://user-images.githubusercontent.com/33333517/180788110-3ab3b0cc-97a0-4954-82e8-7780942b0399.png)
</details>

<details>
  <summary>Coroner 4 - DNR Windoor</summary>
  
  Not sure if intended, but the DNR windoor here has no access requirement. Not only doctors, anyone just slipping in can enter this area.

![image](https://user-images.githubusercontent.com/33333517/180788669-0c70f883-774b-4028-b29f-3f4ba3e6ecc0.png)

</details>

<details>
  <summary>Coroner 5 - Blockable exit</summary>
  
  Not sure if intended, but this tray can be used as a _very_ effective blocker here, to completely block out the northern entrance.

![image](https://user-images.githubusercontent.com/33333517/180788866-312f769a-2246-414f-974c-087daa0d6218.png)

</details>

<details>
  <summary>Coroner 6 - No fire extinguisher cabinet</summary>
  
  Not mandatory, but this place is huge, may want to include one. Or not. Up to you!

</details>

<details>
  <summary>Coroner 7 - Request console</summary>
  
  This request console is at a bit ambiguous spot. I would expect it to be reachable from this spot, yet it is not. You may want to move this or make it accessible from both sides.

![image](https://user-images.githubusercontent.com/33333517/180789198-0614b9e6-9c26-4f53-9e1b-e9f0882226ff.png)
</details>

Miraviel reviewed (changes requested):
CMO office review

<details>
  <summary>CMO Office 1 - Missing camera</summary>
  
  The room has no camera.

  ![image](https://user-images.githubusercontent.com/33333517/180789936-65bff84c-31e9-477b-9d56-108d4fbd863f.png)
</details>

Miraviel reviewed (changes requested):
Internal maintenance

<details>
  <summary>Internal Maintenance 1 - Airlocks accesses</summary>
  
  
Two of these airlocks only need medical access, the other two need medical or construction access. 

![image](https://user-images.githubusercontent.com/33333517/180791317-a26d1108-8d52-4d59-ab36-c58b774ee006.png)

</details>

<details>
  <summary>Internal Maintenance 2 - APCs</summary>
  
  I see you are placing multiple APCs in this small room as an "engineering hub" for the nearby rooms. However, this goes against Paradise's (unwritten!) design that each area should have the APC in the same area it powers. While it is done so area-wise, this will GREATLY confuse engineers. Every other room on this map and every other station has the "APC is in the same room it powers". Except for this little nook. 

  I personally would advise against this for gameplay reasons. If the CMO office is depowered, the engineer should break into the CMO office and fix it there. (And maybe emag the locker in the meantime). The more reasons people should be in areas they should not be, the more possibilities antags have.
  
![image](https://user-images.githubusercontent.com/33333517/180792562-aeb50b6e-1bee-4588-9b6c-3a42d1593b11.png)

</details>

Miraviel reviewed (changes requested):
Medbay

<details>
  <summary>Medbay 1 - Three cameras in one room</summary>
  
  This room has a frightening amount of cameras.

![image](https://user-images.githubusercontent.com/33333517/180793580-5245b284-1850-4c42-84fd-7ec692eb5c3e.png)

</details>

<details>
  <summary>Medbay 2 - Missing firelock</summary>
  
  One missing firelock here.

  
![image](https://user-images.githubusercontent.com/33333517/180793968-2c451f32-c64b-4dcb-98b6-dbaed7168e93.png)

  
</details>

<details>
  <summary>Medbay 3 - Firealarm has no firelocks</summary>
  
  Pulling this firealarm does nothing, no firelocks are connected to it.

  ![image](https://user-images.githubusercontent.com/33333517/180794163-340385c8-8f4e-4637-bb81-a85108f4834a.png)
  
</details>


<details>
  <summary>Medbay 4 - Vents hidden below tables</summary>
  
  There is no rule against this, this is a personal request. Please do not hide atmospheric devices, especially vents under the tables. It is tedious to alt-click them when welding, and the alt-click icon doesn't always update.

  ![image](https://user-images.githubusercontent.com/33333517/180794163-340385c8-8f4e-4637-bb81-a85108f4834a.png)
  
</details>

<details>
  <summary>Medbay 5 - Operating theatre 1 awkward sink, no defib</summary>
  
  The sink in this operating theatre looks... awkward. That hole after the surgical tray + two tables look like something is missing from there. I would suggest moving the sink west of the operating console and placing something there to fill the gap.

  Also, currently we have a defibrillator in the OR by default, it is missing from here.

  ![image](https://user-images.githubusercontent.com/33333517/180794721-03afacf2-1b86-4d2f-a1e6-69405fd4b048.png)
  
</details>

<details>
  <summary>Medbay 6 - Operating theatre 2 no windows, awkward sink, no defib</summary>
  
  This OR has no windows and no window tint control button.
  
  Lacks a defibrillator as well.

  Same awkward corner sink placement, if you decide to change OR1's sink, please change this one's too.
  
![image](https://user-images.githubusercontent.com/33333517/180795199-cb0aac10-5ab0-4ca3-aa5a-6ab9006b1577.png)

</details>

<details>
  <summary>Medbay 7 - Cryotubes and the Main APC</summary>
  
  Placing the oxygen canister this far from the cryotubes is very, very, very prone to get it griefed. From people emptying it to others replacing it with less benevolent canisters and turning up the freezer... I have seen people grief cryotubes before, and this is an invitation for them. If possible, please somehow move the oxytank/freezer to the same room as the cryotube. (I am aware that these are all ahelpable offences, but we should not give admins more work if possible).

  This room also has the APC outside of its actual area (the main corridors). As written above with the internal maintenance, this is very inconsistent to everything we have. If you intended to have a true maintenance room, air alarms should be here as well, but then whoever is in this _windowless_ room has _way_ too much influence over medbay.

  This entire room is an invitation at best, easily mess with medbay, at worst, grief it.

  (Also, one oxytank is enough for two tubes.)

![image](https://user-images.githubusercontent.com/33333517/180795992-b42e1548-2820-4765-a5f0-49267a512168.png)

  
</details>

<details>
  <summary>Medbay 8 - Missing cameras</summary>
  
  These two rooms have no cameras.
 
![image](https://user-images.githubusercontent.com/33333517/180801472-14cdf5d2-e6df-45be-a2be-b7969c69838f.png)

![image](https://user-images.githubusercontent.com/33333517/180801339-c279bf81-5739-4ae3-99a6-efe4ddc04e9b.png)

  
</details>

<details>
  <summary>Medbay 9 - Blood bag freezers</summary>
  
  There is no need to manually map in every blood bag, especially because it is prone to have mistakes. Please use the `/obj/structure/closet/crate/freezer/iv_storage` freezer subtype, it has all the blood (and the here currently missing slime blood) - this way you won't need to map in the vox blood bags either that are currently being added.

  There is no need for oxytanks either, medbay doesn't use these yellow ones, they mostly use the emergency lockers's `/obj/structure/closet/walllocker/emerglocker` blue oxytanks instead.

![image](https://user-images.githubusercontent.com/33333517/180797969-3c36e788-e1d4-4af9-bc35-ed1dceb91da8.png)

![image](https://user-images.githubusercontent.com/33333517/180796917-d4b9e8d1-535a-4013-a739-8f9711157839.png)
  
</details>

<details>
  <summary>Medbay 10 - Lobby has two fire alarms</summary>
  
  There are two fire alarms here (to my east and to my southwest).
  
![image](https://user-images.githubusercontent.com/33333517/180798460-865cb505-fce5-4883-8c70-e16bcb6c46d8.png)

  
</details>

<details>
  <summary>Medbay 11 - Medbay Reception has two fire alarms</summary>
  
  Two fire alarms here, one to my east, one to my northwest.  

![image](https://user-images.githubusercontent.com/33333517/180798702-49e412e7-948a-4884-bdec-008ec367c936.png)

  
</details>

<details>
  <summary>Medbay 12 - Nitpicking about the MediDrobe</summary>
  
  Super nitpicking, but it's strange that the MediDrobe is in the reception, aka a client-facing side of the facility, instead of the locker room (Medical Supplies here), where we have all the gear and mirrors.

  
![image](https://user-images.githubusercontent.com/33333517/180799131-f88e7f2b-e7ae-4043-8bde-ce40c7f65576.png)

</details>

<details>
  <summary>Medbay 13 - Chemistry</summary>
  
  More of a request, please include a patch box as well here, we use as many pills and patches nowadays.

  May also want to add a newscaster to this room as, similarly to virologists, chemists spend most of their time in their workplace by design.
  
![image](https://user-images.githubusercontent.com/33333517/180799583-814822c0-8779-48a9-a56d-62625fe6741c.png)

</details>

S34NW commented:
Thanks for the review @Miraviel it is much appreciated.
I have applied all these changes bar the following:

>CMO Office 1 - Missing camera
>Medbay 6 - Operating theatre 2 no windows
>Medbay 8 - Missing cameras

"Missing" cameras (and windows) are not, in fact missing. These rooms are built in such a way that they are visible by camera normally, but are not visible by camera once the blinds are drawn/windows tinted. It's going to be an experimental quirk for medbay that allows more "privacy".

OR2 is it's whole own discussion that basically boils down to having a more risky OR that antags might decide to use. Why not electrochromics I hear you ask? Well, nosy doctors will notice the windows are dark when they are not normally and go have a look, if the room is always walls then it's not as suspicious and you are less likely to have some nosy git waltz in just because he wants to metagame some antag.

>Medbay 4 - Vents hidden below tables

This is done intentionally (albeit sparingly) across all maps to make vent crawling antag life a little easier.

>Medbay 7 - Cryotubes and the Main APC

Only doctors and engineers can get into this O2 room, it should be fine.

APC is now moved, however that was only the APC for that treatment room, not for the wider medbay as a "main" APC.

S34NW commented:
Thanks, MDB1

Bm0n commented:
I recommend making areaing this place here "coldroom" as seen on Meta

![pr1](https://user-images.githubusercontent.com/92271472/181996170-3b89d650-aba2-4a73-89a5-b70a4822c9e6.png)

Honestly I am not a big fan of blocking off good chunks of maintenance with grillie spawners. I found it was the one big thing that made old Meta maints garbage


![pr2](https://user-images.githubusercontent.com/92271472/181996210-59db10c4-4848-497f-9e13-d7936eea19aa.png)

I feel like maxcaps might below up a bit more of the station now but I am not too sure.

![pr3](https://user-images.githubusercontent.com/92271472/181996221-ab6be7b6-4ceb-4f08-b5df-69f5f1d41679.png)

Also you undid a good chunk of my box refactor. All of the maintenance airlocks have no maintenance access points added to them and there are now lights and cameras back on the glass near escape and in the bomb test range.

sorry I can't get a full review out but I wanted to at least take a small look at this before I leave for vacation

S34NW commented:
> Also you undid a good chunk of my box refactor. All of the maintenance airlocks have no maintenance access points added to them and there are now lights and cameras back on the glass near escape and in the bomb test range.

Thanks for pointing this out, the nature of the large-scale edits this remap involves means a lot of areas just get painted over. I'll go over these and make sure it is all touched up.

S34NW commented:
![image](https://user-images.githubusercontent.com/12197162/181996959-2f4da732-dfe5-412a-89fc-c982bd6725cf.png)
Maxcap range for reference, station is fine.

QuacksQ commented:
![image](https://user-images.githubusercontent.com/79924768/182693095-c751e4a6-f6cc-43da-9aaa-992964d12c7b.png)
This grass tile by xeno bio is kinda scuffed.

Asmerath commented:
![uh](https://user-images.githubusercontent.com/17059084/182719965-704fedfa-3f97-4243-8aa2-fb7a64e44b5e.png)

Found this. didn't get cords but I was maint exploring and someone else and I found this

Sayrbee commented:
Can we get a slight change and move the whole chapel one tile over to the right so the doors of the chapel are lined up with RD?

LightFire53 commented:
<details><summary>*Cracks knuckles*</summary>

### Medical

![image](https://user-images.githubusercontent.com/16112919/182760107-e0d48f0c-e956-46c9-ae85-45894c0d482f.png)

Wonky areaing on airlock

![image](https://user-images.githubusercontent.com/16112919/182760326-d29ded38-8612-49bc-9cbc-c5c4483e133f.png)

Multi-holopad

![image](https://user-images.githubusercontent.com/16112919/182760557-f9adbaca-4a87-41d1-bd83-29a71d2b634f.png)

![image](https://user-images.githubusercontent.com/16112919/182760582-b91b4502-3504-419a-b2e4-d132e00a875a.png)

It feels odd having more security on the medbay facing chem fridge than on the public facing one

![image](https://user-images.githubusercontent.com/16112919/182761409-e1494dfe-6857-40a1-a907-ebffc11005d2.png)

This feels a little messy, having 5 boxes on one rack. Could either space them out across some tables or toss 'em in a locker.

![image](https://user-images.githubusercontent.com/16112919/182761684-26dcf239-528e-4a22-89ca-ba26bee36b3b.png)

6 pen lights within 4 tiles seems a touch excessive, considering doctors spawn with them.

![image](https://user-images.githubusercontent.com/16112919/182761875-e4befebf-3144-45d7-bd59-72175addeab3.png)

We appear to be missing a helmet in storage

![image](https://user-images.githubusercontent.com/16112919/182762345-8f48d796-414d-402a-a235-0b93dfa00a86.png)

Southern airlock could be a maintenance area (keeping the airlock as current) so when rad event occurs, people can <strike>abuse</strike> make use of it

### Science

![image](https://user-images.githubusercontent.com/16112919/182763583-1abef6a9-0e1c-44fd-8209-05b414de8419.png)

Robotics has both robotics and a science requests console

![image](https://user-images.githubusercontent.com/16112919/182763617-55f800ab-d97b-40e0-b9fe-296118ab11ee.png)

They also have twice the amount of welding goggles now, not sure if intentional or not

![image](https://user-images.githubusercontent.com/16112919/182763890-3f9bbdb8-1de2-40b9-95de-2de1b31b62fe.png)

These look really empty due to only having one sheet of glass and metal each. Could probably consolidate to one rack

![image](https://user-images.githubusercontent.com/16112919/182764136-a1ac6316-885b-4bd5-9366-2b38d14da8d2.png)

While I appreciate the lone extinguisher, it feels a little out of place. Could have a wall cabinet or a full fire closet?

### Escape

![image](https://user-images.githubusercontent.com/16112919/182764972-c2c1c060-5807-442c-b4aa-721e607692fa.png)

Escape: Now reducing light pollution by 1000%. (There are no lights in escape except one by a cig vendor)

### Maintenance

![image](https://user-images.githubusercontent.com/16112919/182765270-34b26b32-45ba-4704-b16f-2e04e6c2ca09.png)

While there is a low chance, this are is so crowded that if all the maybe's trigger, it's a pain in the ass to move through since it's all clogged up with welded doors and walls and grills

![image](https://user-images.githubusercontent.com/16112919/182765390-37986d44-a0a5-4f60-870d-c1eb64f67a96.png)

I also wouldn't actually mind seeing the removal of the engineering checkpoint, so it opens up another hallway/corridor to use, or see the checkpoint relocated.

![image](https://user-images.githubusercontent.com/16112919/182765493-4374e44c-46f2-49e5-b51d-57269462f193.png)

This hallway can also easily become cluncky

![image](https://user-images.githubusercontent.com/16112919/182765663-0f9ddd83-b15c-4d14-829d-bcb0264e56c7.png)

With how crowded the rest of maintenance is, this area is really blank

![image](https://user-images.githubusercontent.com/16112919/182765805-b9e9190b-5683-4d89-b570-7a1bec186b5c.png)

Blank spot listed as nearspace

![image](https://user-images.githubusercontent.com/16112919/182765887-316634d5-b65a-4be5-b034-d12d97fe24e3.png)

Rage cage isn't electrified, heresy. I know that it really is out of place of wiring though

Other notes: Liberal use of tinted windows, which I'm not really a huge fan of. They more or less function as a wall except glass, would personally change a few of those to be grill/wall/regular window/girder

I've also noticed that this maintenance now has atmospherics, are we looking at implementing for other maintenance branches?</details>

1080pCat commented:
OR 1 doesn't with a iv drip.
The cyrocells on the left side of medical start with the cooler on, but the cloning side doesn't
There is a door that leads to a wall to the right of OR2. 
![image](https://user-images.githubusercontent.com/96908085/182785778-8bafc864-d586-4bad-b12b-61233160b3d7.png

Outside of the issues above, this new medical is.. good, slightly too big though, takes just a bit too long to get to surgery from the main entry,  The plant segments in the.. _waiting room?_ is aesthetically pleasing, but rather in the way, overall the new medbay does look nice, but slightly too spread out.

S34NW commented:
> There is a door that leads to a wall to the right of OR2.

That is a false wall

Miraviel commented:
More of a request than a bug report, but since we have vox tanks, we could use a spare breath mask or two. I know they can use the regular one, but it is a bit strange.

![image](https://user-images.githubusercontent.com/33333517/182790871-2ce25a9b-ac98-4ce3-876d-17c34f73ecf9.png)

S34NW commented:
> Wonky areaing on airlock

This was an attempt to follow the maint door area principles (Even though it's a storage room not maint). I can revert it if desired.

> It feels odd having more security on the medbay facing chem fridge than on the public facing one

Not much that can be done here, the windoor on the bottom of the medbay one is to prevent theft, but the public one would have to be public and thus defeats the point. This is in-line with the other maps.


> Other notes: Liberal use of tinted windows, which I'm not really a huge fan of. They more or less function as a wall except glass, would personally change a few of those to be grill/wall/regular window/girder

The issues presented with a lot of areas of this map mean there should be "private" spaces or areas hard to see into, but at the same time we want to run pipes or whatever there. There aren't really any options for something that takes a while to deconstruct like tinted windows, as girders are a single action from deconstruction, full walls prevent placements of wires and pipes, and regular windows give vision.


> I've also noticed that this maintenance now has atmospherics, are we looking at implementing for other maintenance branches?

Not planned, this is done here due to the scope of the maints. It would be a herculean task to repressurise with porta-pumps, so this both allows for antag movement and permits resolution to atmos issues within the large area.

Assume anything I have not commented on has been changed.

S34NW commented:
> More of a request than a bug report, but since we have vox tanks, we could use a spare breath mask or two. I know they can use the regular one, but it is a bit strange.

We had these and someone else requested I remove them 😭

S34NW commented:
code change code change scope creep scope creep

S34NW commented on code/modules/research/rdconsole.dm:
```diff
@@ -81,6 +81,7 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 
 	var/id = 0			//ID of the computer (for server restrictions).
 	var/sync = TRUE		//If sync if FALSE, it doesn't show up on Server Control Console
+	var/range = 3		//Range to search for rnd devices in proximity to console
```
```suggestion
	///Range to search for rnd devices in proximity to console
	var/range = 3
```

thanatossage commented:
science doesn't seem to have many fire extinguishers there is 1 up top in a locker and 2 down near xeno no cabinets any where in the hall. that i could see

S34NW commented:
> science doesn't seem to have many fire extinguishers there is 1 up top in a locker and 2 down near xeno no cabinets any where in the hall. that i could see

![image](https://user-images.githubusercontent.com/12197162/182863382-93bc3c01-cbcc-4791-a231-efd16fe4d711.png)

I think there are enough spread about, as illustrated here

LightFire53 commented:
>This was an attempt to follow the maint door area principles (Even though it's a storage room not maint). I can revert it if desired.

If that's the case, that's fine but do it with the other OR as well then

Sadhorizon commented:
RD starts with 2 pairs of superior welding goggles. One on table, one in locker

S34NW commented:
> Found issue, RD starts with two pairs of superior welding goggles. One on table, one in locker

oop, thanks

LightFire53 commented:
Virology is not spawning with viruses in their fridge.
There is no camera in OR 2 or the backroom, not sure if it was intentional or not.

BakKeener commented:
There's a lack of lighting in departures in addition I think you might of accidently touched chapel as their confession booth now has walls right behind the doors.

Coolrune206 commented:
![image](https://user-images.githubusercontent.com/71326864/183246925-b4466b82-6ae4-4319-a2e2-774f02835ad4.png)
The confession booth is, slightly less functional with this remap than before.

Bm0n commented:
You should commit to a single type of window here, blending the two looks weird

![image](https://user-images.githubusercontent.com/92271472/183254129-afa14cd5-2975-4436-bb17-e91c327a8638.png)

Use a single triple book random spawner instead of 3 stacked on top of each other

![image](https://user-images.githubusercontent.com/92271472/183254189-cb0c0972-75d5-4de1-ae2f-4d21aaf8dbda.png)

Another thing I noticed is that toxins just doesn't have an external cooling loop anymore in the mixing room.

LightFire53 commented:
#18732

Taaketa commented:
New Med/Sci maint lacks Crates:

Despite a massive overhaul of Sci/Med Maint there are no crates at all.

What did you expect to happen:

Cargo likes to nom nom crates.

What happened instead:

Some crates to exist in maint areas.

Why is this bad/What are the consequences:

Cargo likes to nom crates for points.

Steps to reproduce the problem:

Go to new Sci/med Maint on Cyberaid and find no crates.

When did the problem start happening:

New Sci/Med Maint.

Extra information:

Stick a few crates in, not a massive amount. Old Sci/med maint had 3 from memory.

AffectedArc07 commented:
This map passes the maptick test

Running a rolling average over the span of 2 weeks for adequate data shows 0 ramp up of MTPC per client on a standard trend line
![image](https://user-images.githubusercontent.com/25063394/183287088-c4181754-74c6-433f-8c6a-53206de66875.png)

*The drop anomalies are when the round is roundstart and that sample is used, everyone is in the lobby and theres 0 map activity* 

I dont do math fully so this data is up for anyone to interpret.

S34NW commented:
> Virology is not spawning with viruses in their fridge.
> There is no camera in OR 2 or the backroom, not sure if it was intentional or not.

Fixed viro, second is intentional. Cam cover for OR2 is fine when windows are clear, added cam on backroom.

> There's a lack of lighting in departures in addition I think you might of accidently touched chapel as their confession booth now has walls right behind the doors.

Thanks, addressed.

> You should commit to a single type of window here, blending the two looks weird

thanks

> Use a single triple book random spawner instead of 3 stacked on top of each other

good shout

> Another thing I noticed is that toxins just doesn't have an external cooling loop anymore in the mixing room.

Good catch, added

> Stick a few crates in, not a massive amount. Old Sci/med maint had 3 from memory.

this map has 3 crates, two south of experimentor and one north-east of the abandoned robo lab

> This map passes the maptick test

praise be

Bm0n commented:
The RD and CMOs office should be surrounded by Rwalls on all sides as it was before and is on other maps.

Bm0n commented:
sorry for another post

This should be off as I changed it in my box refactor to not be dumping perfectly recyclable gasses into space. Besides that it's a var edited pump which is yucky

Turbine pumps are also using varedited on pumps, again something I fixed in my box refactor

![image](https://user-images.githubusercontent.com/92271472/183538844-a50fc485-21d3-4b16-8533-2b7cf431c233.png)

S34NW commented:
Fixed the pump issues

> The RD and CMOs office should be surrounded by Rwalls on all sides as it was before and is on other maps.
> 
> Same could be said about most of the front and sides of science

I disagree here re: head offices, they already have regular reinforced windows so rwalling them entirely is needless. The CMO office is totally surrounded by medbay in this remap (unlike currently), and also has rwalls on the secure storage room side. The RD office does have rwalls on maints-facing walls.

As for science, currently we only rwall potentially dangerous/explosive areas, critical infrastructure, or areas mechs are likely to be stomping about, and the same remains true here. Server room is rwalled, as is experimentor chamber, scichem testing range, xenobio, toxins, toxins storage, public-facing RND walls, and exteruior mech bay walls.

The one part I agree on is adding some more rwalling to is scichem lab, so I did this.

Rwalling off entire departments should be avoided, weak spots are encouraged so that antags dont just ALWAYS use the doors and/or the windows.

Taaketa commented:
What's the reasoning for a seperate vent system?

alex-gh commented:
> Not planned, this is done here due to the scope of the maints. It would be a herculean task to repressurise with porta-pumps, so this both allows for antag movement and permits resolution to atmos issues within the large area.

If the maints are depressurized across multiple sections (that are normally airtight because they are separated by airlocks) it should not be an easy task to repressurize them.
Sure, air pumps are probably insufficient, but atmos techs can always put a T junction to a distro loop and blow in tons of air into maints quickly.

IMO you shouldn't be optimizing jobs away with failsafes like these. Give atmos techs something to do, let them use creativity to solve a potentially difficult problem.

S34NW commented:
> What's the reasoning for a seperate vent system?

If you use the same distro as the rest of the station, vent-crawling antags become much stronger.

alex-gh commented:
> I struggle with this logic because in that case you might as well just remove most of the pumps on the station to give atmos something to do.

Obviously normal operation should not require atmos tech intervention. It is just when things go wrong I'd always err on rewarding player proaction/action/interaction.

Personally I consider redundant and self-healing systems are boring (unless they are player made).

> This maint area is comfortably bigger than medical and science combined, it needs some system for passive refill.

But the area is separated by airlocks. So it would take someone willfully going through the effort of sabotaging the airlocks to stay open to actually depressurize all of it.

Bm0n commented:
I feel like something went wrong here.

![image](https://user-images.githubusercontent.com/92271472/183998008-779370d5-7960-4a19-bec1-09fe3e4d1083.png)

Try this layout. Moving the grille and spaceheaters allows people to strafe into cover from chokepoints/long hallways plus this just cleans up the look of this area.

![fix](https://user-images.githubusercontent.com/92271472/183999713-6ff7bc54-f5e8-495f-96f9-ddca30999f18.png)

besides that I have done my best to look through the rest of the map, looks good to me.

Bm0n commented:
Was opening this up done on purpose? Kind of partly defeats the whole closing blast doors gimmick

![Untitled](https://user-images.githubusercontent.com/92271472/184089861-54612f39-7fda-4f13-83f7-7f56dc9745a4.png)

AffectedArc07 commented:
> Was opening this up done on purpose? Kind of partly defeats the whole closing blast doors gimmick
> 
> ![Untitled](https://user-images.githubusercontent.com/92271472/184089861-54612f39-7fda-4f13-83f7-7f56dc9745a4.png)

Revert this please

AffectedArc07 reviewed (approved):
Ship it
